### PR TITLE
showTemporary instead of show for notifications

### LINKED ISSUE #16495: Invalid storage advice does not dissapear.
### Steps to reproduce
1. Enable option Allow users on this server to send shares to other servers
2. Share a folder by link
3. Unshare the folder
4. Open the fodler in server B

### Expected behaviour
The  advice error should dissapear after surfing the web

### Actual behaviour
The  advice error should keeps until the screen is refreshed

### Server configuration
**Operating system**: ubuntu 14.04

**Web server:** apache 2.4.7

**Database:** mysql

**PHP version:** 5.5.9

**ownCloud version:** "version":"8.1.0.6","versionstring":"8.1 beta 2","edition":"Enterprise"

**Updated from an older ownCloud or fresh install:** Fresh install

**List of activated apps:**
Enabled:
 - activity: 1.2.3
 - admin_audit: 0.6
 - encryption: 1.0.0
 - enterprise_key: 0.1.0
 - files: 1.1.9
 - files_locking: 
 - files_pdfviewer: 0.7
 - files_sharing: 0.6.1
 - files_sharing_log: 0.5
 - files_texteditor: 0.4
 - files_trashbin: 0.6.2
 - files_versions: 1.0.5
 - files_videoviewer: 0.1.3
 - firewall: 0.91.2
 - firstrunwizard: 1.1
 - gallery: 0.6.0
 - provisioning_api: 0.2
 - templateeditor: 0.1
 - updater: 0.4
 - user_ldap: 0.6.0
 - windows_network_drive: 0.1.1

**Are you using external storage, if yes which one:** no

**Are you using encryption:** yes

**Are you using an external user-backend, if yes which one:** ActiveDirectory

### Client configuration
**Browser:** Internet Explorer 9

captura de pantalla 2015-05-21 a las 10 55 21

### PR DESCRIPTION
**Pull request by PVince81**
Fixes https://github.com/owncloud/core/issues/16495

Most error messages in the file list will now disappear after 7 seconds.

@MorrisJobke @blizzz @rullzer @icewind1991

### CODE REVIEW

**Comment by karlitschek** [comment]
:+1:

File: apps/files/js/filelist.js [hunk 1]
```diff
@@ -1640,10 +1640,10 @@
...
 								if (result.status === 'error' && result.data.message) {
-									OC.Notification.show(result.data.message);
+									OC.Notification.showTemporary(result.data.message);
 								}
 								else {
-									OC.Notification.show(t('files', 'Error moving file.'));
+									OC.Notification.showTemporary(t('files', 'Error moving file.'));
 								}
 								// hide notification after 10 sec
 								setTimeout(function() {
```
Comment (nickvergessen) [inline apps/files/js/filelist.js]: remove this?

File: apps/files/js/filelist.js [hunk 2]
```diff
@@ -2013,10 +2013,10 @@
...
 							if (result.status === 'error' && result.data.message) {
-								OC.Notification.show(result.data.message);
+								OC.Notification.showTemporary(result.data.message);
 							}
 							else {
-								OC.Notification.show(t('files', 'Error deleting file.'));
+								OC.Notification.showTemporary(t('files', 'Error deleting file.'));
 							}
 							// hide notification after 10 sec
 							setTimeout(function() {
```
Comment (nickvergessen) [inline apps/files/js/filelist.js]: remove this?

File: apps/files/js/filelist.js [hunk 3]
```diff
@@ -2262,7 +2262,7 @@
 		 */
 		_showPermissionDeniedNotification: function() {
 			var message = t('core', 'You don’t have permission to upload or create files here');
-			OC.Notification.show(message);
+			OC.Notification.showTemporary(message);
 			//hide notification after 10 sec
 			setTimeout(function() {
```
Comment (nickvergessen) [inline apps/files/js/filelist.js]: remove this?

**Comment by PVince81** [comment]
done

**Comment by nickvergessen** [comment]
:+1:

**Comment by icewind1991** [comment]
:+1:

**Comment by davitol** [comment]
:+1: WFM

**Comment by MorrisJobke** [comment]
@PVince81 Please create the backport for 8.2.1

**Comment by PVince81** [comment]
stable8.2: https://github.com/owncloud/core/pull/19774

**Comment by karlitschek** [comment]
please backport :+1:

**Pull request merged**